gimme

```diff
@@ -611,16 +611,16 @@ _resolve_version() {
 	local base="${1%.x}"
 	local ver last='' known
 	known="$(_update_remote_known_list_if_needed)" # will be version-sorted
+	if [[ ! "${base}" =~ ^[0-9.]+$ ]]; then
+		warn "resolve pattern '${base}.x' invalid for .x finding"
+		return 2
+	fi
+	# The `.x` is optional; "1.10" matches "1.10.x"
```

Comment:
This seems kind of strange -- do we support `1.10.0` for users to get the actual upstream 1.10 release? (upstream doesn't do `.0` releases, so while I agree that users would expect `1.10` to give them the same thing as `1.10.x`, I don't think that's correct unless we add our own explicit support for a `.0` suffix to mean "the first `1.10` release)

Reply:
The `.x` means "whatever goes here", where "whatever" might be "nothing".

We expect folks to be able to enter `1.42.x` when `1.42` comes out so as to stay up-to-date on that release series.  We have no other expected syntax to indicate this, since we don't have operators, only version numbers/patterns.

Reply:
Ah, I get it now 👍

I had misread that as a user specifying `1.10` being the same as `1.10.x` but it's instead that the latter will match the former.

LGTM